.project

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This file should be removed from the image - in general we do not commit IDE packaging files from particular environments